lib/transformations/outputPath/__testfixtures__/outputPath-1.input.js

```diff
@@ -0,0 +1,6 @@
+const path = require('path');
+module.exports = {
+	output: {
+		path: path.join(__dirname, 'dist')
```

Comment:
Should we add a test where `path` is imported as a variable with a different name?

`const p = require('path')` should also work I guess.